.github/workflows/proofing-deploy.yaml

```diff
@@ -0,0 +1,45 @@
+name: Deploy proofing site to dhq-proofing.digitalhumanities.org
+
+on:
+  # push:
+  #   branches: [ "main" ]
```

Comment:
I think it'd be useful to have the deploy happen automatically when commits are pushed to `main`!